GVFS/GVFS.Service/Mac/GVFSMountProcess.cs

```diff
@@ -0,0 +1,47 @@
+﻿using GVFS.Common;

```

Comment:
This class is the same as the one in `GVFS.Service.Windows` project *minus* `CurrentUser` and `PrjFlt` related methods.

Reply:
Good suggestion, I'll try that.

Reply:
On second thought - this class is going to be very different on Mac. New `GVFS.Mount` processes will be launched in the same user session. There is not going to be much re-usability in having a common base class.